tardis/visualization/plot_util.py

```diff
@@ -574,3 +559,39 @@ def extract_spectrum_data_hdf(hdf, packets_mode):
             hdf[f"{spectrum_prefix}/wavelength"].to_numpy(), "cm"
         ).to("AA"),
     }
+
+
+def create_wavelength_mask(
```

Comment:
Add demo for this in notebook

Reply:
https://github.com/tardis-sn/tardis/pull/3085